examples/rf.py

```diff
@@ -45,36 +49,19 @@ def rfr(cfg, seed):
         bootstrap=cfg["do_bootstrapping"],
         random_state=seed)
 
-    rmses = []
-    for train, test in kf:
-        # We iterate over cv-folds
-        X_train, X_test = X[train], X[test]
-        y_train, y_test = y[train], y[test]
-
-        rfr.fit(X_train, y_train)
-
-        y_pred = rfr.predict(X_test)
-
-        # We use root mean square error as performance measure
-        rmse = np.sqrt(np.mean((y_pred - y_test)**2))
-        rmses.append(rmse)
-    return np.mean(rmses)
+    def rmse(y, y_pred):
+        return np.sqrt(np.mean((y_pred - y)**2))
```

Comment:
I don't think so, since it returns a single float right now which is interpreted by the cross_val_score. I chose this over the old version for readability, would you prefer the old (explicit) version?